packages/apple-pay-integration/src/ApplePayPaymentMethod.tsx

```diff
@@ -0,0 +1,43 @@
+import { toResolvableComponent, PaymentMethodProps } from '@bigcommerce/checkout-js/payment-integration';
+import React, { FunctionComponent, useEffect } from 'react';
+
+const ApplePaymentMethod: FunctionComponent<PaymentMethodProps> = ({ method, checkoutService, language, onUnhandledError }) => {
+    useEffect(() => {
+        const componentDidMount = async ():Promise<void> => {
+            try {
+                await checkoutService.initializePayment({
+                    gatewayId: method.gateway,
+                    methodId: method.id,
+                    applepay: {
+                        shippingLabel: language.translate('cart.shipping_text'),
+                        subtotalLabel: language.translate('cart.subtotal_text'),
+                    },
+                });
+            } catch (error) {
+                onUnhandledError(error);
+            }
+        }
+        componentDidMount().then();
```

Comment:
Do we need `.then` here, should be ok to do `componentDidMount()`

Reply:
The functionality would stay the same, but it can help to get rid of the eslint warning, `Promise returned from componentDidMount is ignored`.

Reply:
why not just do eslint ignore comment?

Reply:
I have removed `.then()` as we actually intend to ignore that non-critical warning.